core/src/main/java/org/jdbi/v3/ResultBearing.java

```diff
@@ -24,6 +31,8 @@
     @Override
     ResultIterator<ResultType> iterator();
 
+    ContainerFactoryRegistry getContainerFactoryRegistry();
```

Comment:
I'm not sure I like exposing the registry in the ResultBearing interface. We try to keep these classes internal so they can evolve without breaking compatibility.